dflow-demo/src/environments/environment.prod.ts

```diff
@@ -1,3 +1,4 @@
 export const environment = {
-  production: true
+  production: true,
+  API_URL: "https://devex-von-permitify-dev.pathfinder.gov.bc.ca/bc-tob/"
```

Comment:
I am not sure how to get this to dynamically point to the test instance for the test deployment. @cywolf ideas?

Reply:
Nevermind, I updated the configurations in a way that seems to work in Docker, and therefore *should* work in OpenShift (will test once deployed).